pkg/cli/topo.go

```diff
@@ -309,53 +287,60 @@ func watch(cmd *cobra.Command, args []string, filter string) error {
 		return err
 	}
 
-	updatec := make(chan *topo.Update)
+	updates := make(chan *topo.Update)
 
-	go watchStream(stream, updatec)
+	go watchStream(stream, updates)
 
-	watchReport(updatec, filter)
+	printIt(updates, objectType, nil)
 
 	return nil
 }
-func watchStream(stream topo.Topo_SubscribeClient, updatec chan *topo.Update) {
+func watchStream(stream topo.Topo_SubscribeClient, updates chan *topo.Update) {
 	for {
 		response, err := stream.Recv()
 		if err == io.EOF {
 			// read done.
-			close(updatec)
+			close(updates)
 			return
 		}
 		if err != nil {
 			cli.Output("Error receiving notification : %v", err)
-			close(updatec)
+			close(updates)
 			return
 		}
 
 		for _, update := range response.Updates {
-			updatec <- update
+			updates <- update
 		}
 	}
 }
 
-func watchReport(updatec chan *topo.Update, filter string) {
+func printIt(updates chan *topo.Update, objectType topo.Object_Type, done chan bool) {
```

Comment:
The response to the `get` has a first column which is the Update Type - this should not be shown for `get` because these are just in the database, and are not updates (that's only for watch)

In the output, there are no headers, and the columns don't align e.g.

```
$ go run ./cmd/onos topo get entities --service-address localhost:5150
UNSPECIFIED   ENTITY   ric0   RIC
UNSPECIFIED   ENTITY   ric1   RIC
UNSPECIFIED   ENTITY   e2node1   E2NODE
UNSPECIFIED   ENTITY   e2node2   E2NODE
```